scrapers/StudioFOW.yml

```diff
@@ -1,3 +1,7 @@
+# History:
+# Dec 2020: created by bnkai
+# July 2022: updated by erri120
+
```

Comment:
We do not add history in any of the yml files to keep things clear.
Edit history is available anyway from the github repo (you can `blame` for example the file)